custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -181,6 +182,7 @@ class ArgumentsOptions:
     CLASS_LABELS_FILE = "--class-labels-file"
     SKIP_DEPS_INSTALL = "--skip-deps-install"
     SPARSE_COLFILE = "--sparse-column-file"
+    PARAMETER_FILE = "--parameter-file"
```

Comment:
You may want to add PARAMETER_FILE into `ArgumentOptionsEnvVars`, so it will be automatically taken by DRUM if you provide `PARAMETER_FILE` env var. 
It is useful in the env's `test_fit.sh` so you won't need to read env var and pass it into --parameter-file